src/components/commons/inputs/ImageField.module.scss

```
@@ -3,6 +3,16 @@
 
   &-label {
     @include text-style(14, $white);
+
+    &-container {
+      @include flexbox($gap: 0.8rem);
+    }
+
+    &-description {
+      @include text-style(10);
+
+      color: $gray40;
```

```suggestion
      @include text-style(10, $gray40);
```
으로 믹스인 사용하시면 좋을 것 같습니다~